### PR TITLE
Add mt-metadata

### PR DESCRIPTION
**Pull request by NCI-datacollections**
Establishing a new W3ID persistent URI namespace for the mt-metadata standard.

### CODE REVIEW

**Comment by NCI-datacollections** [comment]
Thank you @dgarijo!

File: mt-metadata/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+**NCI Data Collections**   
+National Computational Infrastructure (https://nci.org.au)
+Canberra, Australia
+<datacollections.nci@anu.edu.au>
+GitHub: [@NCI-datacollections](https://github.com/NCI-datacollections)
+
+**NCI Help**  
+National Computational Infrastructure (https://nci.org.au)
+Canberra, Australia
+<help@nci.org.au>
```
Comment (TallTed) [inline mt-metadata/README.md:27]: to preserve apparently intended line breaks --
```suggestion
**NCI Data Collections**  
National Computational Infrastructure (https://nci.org.au)  
Canberra, Australia  
<datacollections.nci@anu.edu.au>  
GitHub: [@NCI-datacollections](https://github.com/NCI-datacollections)

**NCI Help**  
National Computational Infrastructure (https://nci.org.au)  
Canberra, Australia  
<help@nci.org.au>
```

Comment (NCI-datacollections) [reply mt-metadata/README.md:27]: Thanks @TallTed. That looks fab.

Comment (TallTed) [reply mt-metadata/README.md:27]: Note that you'll need to submit another PR to get these into place.

**Pull request merged**